Tests/DatadogTests/Datadog/DatadogCore/DatadogCoreTests.swift

```diff
@@ -9,12 +9,13 @@ import TestUtilities
 import DatadogInternal
 @testable import Datadog
 
-private struct FeatureMock: DatadogFeature {
+private struct FeatureMock: DatadogProduct {
```

Comment:
nit/ Quite hard to figure out this lingo - `FeatureMock` that is a `DatadogProduct` in fact (whereas `DatadogFeature` is sth else).

Reply:
I will rename `DatadogProduct` to `DatadogRemoteFeature`, which is more representative of its capability. 'Product' is even more confusing,